stack.cabal

```diff
@@ -49,6 +49,10 @@ flag static
   -- Not intended for general use. Simply makes it easier to
   -- build a fully static binary on Linux platforms that enable it.
 
+flag use-old-locale
+  default:     False
+  description: Use old-locale instead of time >= 1.5
```

Comment:
As said in #2377, I just didn't realize there was a `stack-7.8.yaml`, so this addition is rather useless. I'll remove it in a moment.